### ISSUE TITLE
???? looking at the .py code, 

### ISSUE THREAD

**Issue by Elnuevocrack** [ch0l0hatwhite/kill-whatsapp-#1]
This git is a very good way to run out of a direct path to phone files, and no phone files at the same time.
The "rm -rf" command delete phone main directory without consulting the user, and he does nothing of what he promises.

**Comment by ch0l0hatwhite** [comment]
correct, is a joke for the script kiddies to he learn a leasson to read code before to execute
